falcon_multiqc/commands/save.py

```diff
@@ -21,156 +22,182 @@
     batch_description {string} -- Set this if the input is 1 batch.
     cohort_description {string} -- Set this if the input is 1 cohort.
     batch_metadata {file} -- A csv with header "Batch Name,Description". Set this if the input is multiple batches.
+    csv {file} -- a csv with directories and sample_metadata for bulk saving. If this option is used --directory and --sample_metadata are not required.
 
 """
 
+
+def save_sample(directory, sample_metadata, session, cohort_description, batch_description):
+    """Saves one result directory and sample_metadatadata to the falcon_multiqc database"""
+
+    # TODO change this to the dirname/basename stuff with os.path
```

Comment:
What do you mean by these todos?

Reply:
whoops ive already done those things - thanks for flagging 👍